packages/pluggableWidgets/accordion-web/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this widget will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [unreleased]
+### Added
+- We added a widget property to configure the render mode of the header text.
+- We added widget properties to influence the initial collapsed state for a group.
+- We added a widget properties to control the collapse state of a group via an entity attribute.
+
```

Comment:
In the PR comment you mentioned "Add consistency check for single expanded mode & multiple start expanded configuration.", which is not here. Is that expected?

Reply:
Yes, it's an implementation detail that isn't very important for users. It's basically covered by line 9.

Reply:
Ahh I understand, fine with me!